docker-compose.yml

```diff
@@ -18,6 +18,8 @@ services:
       FHIR_URI: "http://fhir-api:8080"
       ROLE_SERVICE_URI: "http://role-service:82"
       SITE_SERVICE_URI: "http://site-service:83"
+      SPRING_CLOUD_CONFIG_ENABLED: "false"
```

Comment:
these should be defined in the local bootstrap....i.e. the local porofile will define this

Reply:
Done